charts/victoria-metrics-distributed/templates/per-az/vmauth-write-balancer.yaml

```diff
@@ -7,10 +7,14 @@ apiVersion: operator.victoriametrics.com/v1beta1
 kind: VMAuth
 metadata:
   {{- $_ := set $ctx "appKey" (list "zone" "write" "vmauth" "spec") }}
+  {{- if ($zone.write).vmauth }}
+    {{- $urlSpec := $zone.write.vmauth.spec | default dict }}
+    {{- $_ := set $zone.write.vmauth "spec" (mergeOverwrite (deepCopy $.Values.common.vmauth.spec) $urlSpec) }}
```

Comment:
It's already done in below, no?
https://github.com/VictoriaMetrics/helm-charts/blob/fc21494fc968d58714940dbe4f968ca27d8c9706/charts/victoria-metrics-distributed/templates/per-az/vmauth-write-balancer.yaml#L22